src/index.js

```diff
@@ -1,2 +1,34 @@
 import './assets/style.css';
-import './modules/payer.js';
\ No newline at end of file
+import API from './modules/api.js';
+
+const nameElement = document.getElementById('your-name');
+const scoreElement = document.getElementById('your-score');
+const btnRefresh = document.querySelector('.btn-refresh');
+const btnSubmit = document.querySelector('.btn-submit-score');
+const leadersList = document.querySelector('.leader-view-list');
+const api = new API();
+
+const displayLeaders = async () => {
+  const scoresArray = await api.getScores();
+  let scoresData = '';
+  const sortedScores = scoresArray.result.sort((a, b) => b.score - a.score);
+  sortedScores.forEach((item) => {
+    scoresData += `<li>${item.user}: ${item.score}</li>`;
+  });
+  leadersList.innerHTML = scoresData;
+};
+
+btnRefresh.addEventListener('click', async () => {
+  displayLeaders();
+});
+
+btnSubmit.addEventListener('click', async (e) => {
+  e.preventDefault();
+  const playerName = nameElement.value;
+  const playerScore = scoreElement.value;
+  await api.addLeader(playerName, playerScore);
+  nameElement.value = '';
+  scoreElement.value = '';
+});
```

Comment:
👏🎉 Good job so far! Your project is coming along nicely. 🌟 However, there's a small improvement you can make to enhance the user experience.

- [ ] [Optional]💡🔄 Currently, after clicking the submit button, the user needs to manually refresh the page to update the score list. But you can make it more user-friendly! 😊 After successfully adding the value to the API, you can call your `displayLeaders()` function to automatically update the score list and display the new entry. This way, the user won't have to go through the extra step of refreshing the page.
```suggestion
btnSubmit.addEventListener('click', async (e) => {
  e.preventDefault();
  const playerName = nameElement.value;
  const playerScore = scoreElement.value;
  await api.addLeader(playerName, playerScore);
  nameElement.value = '';
  scoreElement.value = '';
  displayLeaders();
});
```
🚀💻 By implementing this enhancement, you'll make your project even more seamless and user-friendly. Keep up the great work, and good luck with the rest of your project! You're doing fantastic! 💪🎉